lib/src/components/NxFilterInput/NxFilterInput.tsx

```diff
@@ -17,7 +17,7 @@ export { Props } from './types';
 const NxFilterInput = forwardRef<HTMLDivElement, Props>(
     function NxFilterInput(props, ref) {
       const { value, placeholder, onChange, className, inputId, disabled, list, ...otherProps } = props,
-          classes = classnames('nx-filter-input', className, {
+          classes = classnames('nx-filter-input', 'nx-text-input', className, {
```

Comment:
This adds `width: 300px` which contradicts the Zeplin files which show the filters has having a width of `250px`.

Reply:
> This adds width: 300px which contradicts the Zeplin files which show the filters has having a width of 250px.

@dturner32 says the 250 only applies in the table filter header.  Implemented in 7cdb2ee4